modules/oadp-storage-class-mapping.adoc

```diff
@@ -11,6 +11,6 @@ Storage class mapping allows you to define rules or policies specifying which st
 You can use the `change-storage-class-config` field to change the storage class of your data objects, which lets you optimize costs and performance by moving data between different storage tiers, such as from standard to archival storage, based on your needs and access patterns.
 
 [id=storage-class-mapping-mtc_{context}]
-== Storage class mapping with MTC
+== Storage class mapping with {mtc-short}
```

Comment:
Would this be better: Storage class mapping with Migration Toolkit for Containers
```suggestion
== Storage class mapping with {mtc-full}
```